style.css

```diff
@@ -0,0 +1,1677 @@
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .first-page {
+    display: none;
+  }
+
+  .desk-first-page {
+    display: flex;
+    min-height: 90vh;
+    background: #1c1a19;
+    background-image: url("./assets/images/Gourp1-desktop.png");
+    background-repeat: no-repeat;
+    background-attachment: inherit;
+    background-position: right top;
+    background-size: 35%;
+  }
+
+  .desk-headline {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 50vw;
+  }
+
+  .desk-hey {
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+  }
+
+  .desk-softwaredev {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+  }
+
+  .desk-headline-p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .socila-link {
+    display: flex;
+    align-self: center;
+    margin-top: 120px;
+    width: 5vw;
+    flex-direction: column;
+    order: -1;
+  }
+
+  /* Cards */
+
+  .cards {
+    display: none;
+  }
+
+  /* desk popup start */
+  .modalD {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 80%;
+    height: 80%;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+    margin: 80px;
+  }
+
+  /* Modal Content */
+
+  .modal-contentD {
+    display: flex;
+    flex-direction: column;
+    background: #fff;
+    padding: 10px;
+  }
+
+  .modalHead-closeD {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 95%;
+    padding: 16px;
+  }
+
+  .modalHeaderD {
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  /* The Close Button */
+
+  .closeD {
+    color: #aaa;
+    font-size: 37px;
+    font-weight: bold;
+  }
+
+  .closeD:hover,
+  .closeD:focus {
+    color: #000;
+    text-decoration: none;
+    cursor: pointer;
+  }
+
+  .desk0 {
+    display: flex;
+    flex-direction: column;
+    margin-top: 2px;
+    padding-left: 20px;
+  }
+
+  #modalTech {
+    display: flex;
+    list-style: none;
+    gap: 7px;
+    margin-top: 0;
+    padding-bottom: 10px;
+  }
+
+  .modalTechD > li {
+    border: 1px solid gray;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #3c3a39;
+    padding: 10px;
+  }
+
+  .desk1 {
+    display: flex;
+    gap: 20px;
+    margin-bottom: 20px;
+  }
+
+  .desk1 > img {
+    width: 60%;
+    height: auto;
+  }
```

Comment:
- [ ] Please make sure the image for the project card takes up more space as shown in the Figma design below. 👇
- Figma design:
  ![image](https://github.com/solog0039/Portfolio-final/assets/104968236/7384d8c0-c079-44c8-825c-4c8c1a6fd570)
- Your design:
  ![image](https://github.com/solog0039/Portfolio-final/assets/104968236/9e4b815f-c1af-4945-b42d-d09d649f7b5a)